packages/web/src/pages/upload-page/fields/StemsAndDownloadsMenuFields.tsx

```diff
@@ -58,7 +58,9 @@ const messages = {
     `Price must be at least $${formatPrice(minPrice)}.`,
   priceTooHigh: (maxPrice: number) =>
     `Price must be less than $${formatPrice(maxPrice)}.`,
-  noDownloadableAssets:
+  losslessNoDownloadableAssets:
+    'You must enable full track download or upload a stem file to provide lossless files.',
```

Comment:
Weird that this message is missing "the" before "full track" while the next message has it..

Reply:
ha good eyes. looks like copy is consistent in design. maybe was recently updated. handled!